pkg/rke/k8s_version_info.go

```diff
@@ -45,16 +45,18 @@ func loadRancherDefaultK8sVersions() map[string]string {
 		"2.7.2":        "v1.25.x",
 		"2.7.3":        "v1.25.x",
 		"2.7.4":        "v1.25.x",
+		"2.7.5":        "v1.26.x",
+		"2.7.6":        "v1.26.x",
```

Comment:
I gather these are technically not needed for 2.8 KDM branch but it was missed in the original PR - https://github.com/rancher/kontainer-driver-metadata/pull/1235/files#diff-6265db4bf76eb753a17a5a74e50dc12ece59fc1638c916628ffd10b2ef8076b7R48-R49

If this is something that's important we will need other 2.7.x versions added there but I don't think so. @kinarashah , please confirm.

Reply:
Yep @snasovich, we don't need these for v2.8.0 branch since this will be referenced only by 2.8 versions. (Though we do try to keep this version info consistent between different branches)